Sources/TuistCore/Descriptors/CommandDescriptor.swift

```diff
@@ -6,12 +6,24 @@ import Foundation
 /// generating a project or workspace.
 ///
 /// - seealso: `SideEffectsDescriptor`
-public struct CommandDescriptor: Equatable {
+public struct CommandDescriptor: Equatable, Hashable, CustomStringConvertible {
     public var command: [String]
 
     /// Creates a command descriptor
     /// - Parameter command: The command and its arguments to perform
     public init(command: [String]) {
         self.command = command
     }
+
+    // MARK: - Hashable
+
+    public func hash(into hasher: inout Hasher) {
```

Comment:
there isn't a need to manually write this out - the compiler should be able to synthesize this automatically :)

Reply:
(the same with the other `hash(into` methods